CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased](https://github.com/o1-labs/o1js/compare/fb625f...HEAD)
 
+### Added
+
+- `setVerificationKeyUnsafe` static method to `SmartContract`
```

Comment:
fyi we usually try to link to the PR as well https://github.com/o1-labs/o1js/pull/2091/